lib/test-utils/cardano-wallet-test-utils.cabal

```diff
@@ -35,7 +35,7 @@ library
     , contra-tracer
     , filepath
     , file-embed
-    , generic-lens >=1.1.0.0 && < 1.2.0.0
+    , lens
```

Comment:
I'd rather not depend on `lens` for this though. The `generic-lens` is a simpler alternative. If the problem regards the upper-bound on the library, then let's work that out.

Reply:
@KtorZ generic-lens 2.0.0.0 have pretty big incompatible changes., plus don't provide `(^?)`

Reply:
(^?) is syntactic sugar, we can live without or re-implement it if needed. 

We really use a small part of the generic-lens API so I imagine the changes to be somewhat manageable. We could also consider `microlens` if it turns out to be too complicated.

Reply:
> (^?) is syntactic sugar, we can live without or re-implement it if needed.
+1 for this

> We really use a small part of the generic-lens API so I imagine the changes to be somewhat manageable. We could also consider `microlens` if it turns out to be too complicated.

microlens dont have Prism :(

Reply:
It has _Left and _Right Traversals which if I recall correctly is the only thing we use Prism for in this package, so it's kinda equivalent.

Reply:
Actually, it's only used in `countMsg` and it seems like this is now dead-code. The function isn't used anywhere anymore so we may as well get rid of it and the dependency altogether.

Reply:
> Actually, it's only used in `countMsg` and it seems like this is now dead-code. The function isn't used anywhere anymore so we may as well get rid of it and the dependency altogether.

Best PR one can get :) Removing unnecessary code is always good! Glad you guys found that out!

Reply:
I'll have to re-create the PR though, our CI pipeline does not run for PR made from external repositories.